src/main/target/common_pre.h

```diff
@@ -330,6 +330,10 @@
 #define USE_THRUST_LINEARIZATION
 #define USE_TPA_MODE
 
+#if defined(USE_SERIALRX_SRXL2) && !defined(USE_SERIALRX_SPEKTRUM)
```

Comment:
Is common_pre.h good place? it will be hard to modify in target.h.

But that would mean moving #ifdef USE_SERIALRX_SPEKTRUM (below) too and that would complicate things even mode.